### PR TITLE
Trim quotes on event properties

### PR DESCRIPTION
**Pull request by EDsCODE**
## Changes

*Please describe.*  
*If this affects the front-end, include screenshots.*  

## Checklist

- [ ] All querysets/queries filter by Team (if this PR affects any querysets/queries)
- [ ] Backend tests (if this PR affects the backend)
- [ ] Cypress E2E tests (if this PR affects the front and/or backend)

### CODE REVIEW

File: ee/clickhouse/models/event.py [hunk 1]
```diff
@@ -105,6 +109,13 @@ def get_elements_hash(self, event):
         return event[6]
 
 
+def _unpad_property(val: str):
+    if val.isdigit() or val == "true" or val == "false":
```
Comment (macobo) [inline ee/clickhouse/models/event.py:113]: Isn't this if clause redundant?

`'true'.strip('"') => 'true'`

File: ee/clickhouse/models/event.py [hunk 2]
```diff
@@ -80,9 +80,13 @@ def get_id(self, event):
...
         if len(event) >= 10 and event[8] and event[9]:
-            return dict(zip(event[8], event[9]))
+            prop_vals = [_unpad_property(res) for res in event[9]]
+            return dict(zip(event[8], prop_vals))
         else:
-            return json.loads(event[2])
+            props = json.loads(event[2])
+            for key, value in props.items():
+                props.update({key: _unpad_property(value)})
+            return props
```
Comment (macobo) [inline ee/clickhouse/models/event.py:89]: Nitpick: Can be shortened to `{ key: _unpad_property(value) for key, value in props.items() }`

This should be slightly faster than updating dict key by key.

**Review by macobo — CHANGES_REQUESTED** [review]

File: ee/clickhouse/models/event.py (hunk 1 above)
Comment (EDsCODE) [reply ee/clickhouse/models/event.py:113]: oh yes it doesn't need the true or false check here!

File: ee/clickhouse/models/event.py (hunk 2 above)
Comment (EDsCODE) [reply ee/clickhouse/models/event.py:89]: got it thanks!

File: ee/clickhouse/models/event.py (hunk 1 above)
Comment (EDsCODE) [reply ee/clickhouse/models/event.py:113]: in fact no check is needed. I was confusing myself with the reverse of this function that I had to do

**Review by macobo — APPROVED** [review]

**Pull request merged**